packages/esm-service-queues-app/src/active-visits/active-visits-row-actions.component.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { FC } from 'react';
+import TransitionMenu from '../queue-entry-table-components/transition-entry.component';
+import { type QueueTableCellComponentProps, type QueueTableColumn } from '../types';
+import ActionsMenu from '../queue-entry-table-components/actions-menu.component';
+import EditMenu from '../queue-entry-table-components/edit-entry.component';
+import { useConfig } from '@openmrs/esm-framework';
+import { type ConfigObject } from '../config-schema';
+import { mapVisitQueueEntryProperties } from './active-visits-table.resource';
+import styles from './active-visits-row-actions.scss';
+
+export const ActiveVisitRowActionsCell = ({ queueEntry }: QueueTableCellComponentProps) => {
```

Comment:
Table column definition containing actions user can perform to each row.

Reply:
We probably havne't been doing this much elsewher,e but might as well add this as a comment above the component definition itself?